bcManager/bcManager.py

```diff
@@ -1951,13 +1996,13 @@ def get_player_data_from_replay_by_plat_id(self, replay_json, platform, platform
                     return player
         return {}
     
-    # TODO: can we make this async AND have it be referenced with a to_thread call?
-    def get_latest_player_data_by_platform_name(self, guild, platform, plat_name):        
+    async def get_latest_player_data_by_platform_name(self, guild, platform, plat_name):        
+        """ Get latest player data by platform name """
         bapi : ballchasing.Api = self.ballchasing_api[guild]
         
         data = bapi.get_replays(sort_by="replay-date", sort_dir="desc", player_name=plat_name, uploader=RSC_STEAM_ID, count=10)
```

Comment:
Was written before but can rename to `replays` for consistency here

Reply:
There are multiple types of data pulls by id or name. Until I come up with a better naming solution to take over from the old one I'd like to keep using it.